CareKitUI/CareKitUI/Components/TaskCategory/Button/OCKTaskCategoryButton.swift

```diff
@@ -0,0 +1,153 @@
+//
+/*
+ Copyright (c) 2020, Apple Inc. All rights reserved.
+ 
+ Redistribution and use in source and binary forms, with or without modification,
+ are permitted provided that the following conditions are met:
+ 
+ 1.  Redistributions of source code must retain the above copyright notice, this
+ list of conditions and the following disclaimer.
+ 
+ 2.  Redistributions in binary form must reproduce the above copyright notice,
+ this list of conditions and the following disclaimer in the documentation and/or
+ other materials provided with the distribution.
+ 
+ 3. Neither the name of the copyright holder(s) nor the names of any contributors
+ may be used to endorse or promote products derived from this software without
+ specific prior written permission. No license is granted to the trademarks of
+ the copyright holders even if such marks are included in this software.
+ 
+ THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+ FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import UIKit
+
+open class OCKTaskCategoryButton: OCKAnimatedButton<OCKStackView> {
+    // MARK: Properties
+
+    /// Determines the label text and image shown in the button.
+    public enum `Type`: String {
+
+        /// Button for phone calls.
+        case call = "Call"
+
+        /// Button for text messages.
+        case message = "Message"
+
+        /// Button for emails.
+        case email = "E-mail"
+
+        var image: UIImage? {
+            let image: UIImage?
+            switch self {
+            case .call: image = UIImage(systemName: "phone")
+            case .message: image = UIImage(systemName: "text.bubble")
+            case .email: image = UIImage(systemName: "envelope")
+            }
+            return image
+        }
+    }
```

Comment:
Why does the task category button have this code in it?

Reply:
Let me remove the whole class, I don't think we'll need such a button.